src/Common/Commands.Common/AzurePSCmdlet.cs

```diff
@@ -58,6 +60,28 @@ protected virtual bool IsErrorMetricEnabled
         }
 
         /// <summary>
+        /// Indicates installed PowerShell version
+        /// </summary>
+        private string _psVersion;
+
+        /// <summary>
+        /// Get PsVersion returned from PowerShell.Runspace instance
+        /// </summary>
+        protected string PSVersion
+        {
+            get
```

Comment:
This value is available from Cmdlet.CommandRuntime.Host.Version.  You will need to make sure that Host is not null.